docs/standard/io/file-path-formats.md

```diff
@@ -0,0 +1,171 @@
+
+## Traditional DOS paths
+
+A standard DOS path can consist of three components:
+
+- A volume or drive letter followed by the volume separator (`:`).
+- A directory name. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates subdirectories within the nested directory hierarchy.
+- An optional filename. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates the file path and the filename.
+
+If all three components are present, the path is absolute. If no volume or drive letter is specified and the directory names begins with the [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>), the path is relative from the root of the current drive is assumed. Otherwise, the path is relative to the current directory. The following table shows some possible directory and file paths.
+
+|Path  |Description  |
+| -- | -- |
+| C:\\Documents\Newsletters\Summer2018.pdf | An absolute file path from the root of drive C: |
+| \\Program Files\\Custom Utilities\\StringFinder.exe | An absolute path from the root of the current drive. |
+| 2018\\January.xlsx | A relative path to a file in a subdirectory of the current directory. |
+| ..\\Publications\\TravelBrochure.pdf | A relative path to file in a directory that is a peer of the current directory. |
+
+## UNC paths
+
+Universal naming convention (UNC) paths, which are used to access network resources, have the following format:
+
+- A server or host name, which is prefaced by \\\\. The server name can be a NetBIOS machine name or an IP/FQDN address (IPv4 as well as v6 are supported).
+- A share name, which is separated from the host name by \\. Together, the server and share name make up the volume.
+- A directory name. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates subdirectories within the nested directory hierarchy.
+- An optional filename. The [directory separator character](<xref:System.IO.Path.DirectorySeparatorChar>) separates the file path and the filename.
+
+The following are some examples of UNC paths:
+
+|Path  |Description  |
+| -- | -- |
+| \\\\system07\\C$\\ | The root directory of the C: drive on `system07`. |
+| \\Server2\\Share\\Test\\Foo.txt | The Foo.txt file in the Test directory of the \\\\Server2\\Share volume.|
+
+UNC paths are always absolute. You can use relative paths only by mapping a UNC path to a drive letter.
+
+## DOS device paths
+
+The Windows operating system has a unified object model that points to all resources, including files. These object paths are not directly accessible from the Windows APIs (such as the Console windows and Windows Explorer). They are, however, exposed to the Win32 layer through a special folder of symbolic links that legacy DOS and UNC paths are mapped to. This special folder is accessed via the DOS device path syntax, which is one of:
```

Comment:
They are accessible from the CMD prompt.

Reply:
In RS4 there is at least some support for device paths in the File Explorer as well. You can type `\\?\C:\Users` in the address bar in File Explorer, the search bar, run prompt, etc.  I'd not mention where things are supported or at what level outside of .NET.